v2/concordium/types.proto

```diff
@@ -1640,13 +1722,27 @@ message CommissionRanges {
   InclusiveRangeAmountFraction transaction = 3;
 }
 
-// A bound on the relative share of the total staked capital that a baker can
-// have as its stake. This is required to be greater than 0.
+// The capital bound is a chain parameter that is set to guarantee network decentralization by preventing 
+// a single validator from gaining excessive power in the consensus protocol.
+// The capital bound is required to be set to a value greater than 0 (capital_bound > 0).
+// The value roughly describes the maximum proportion of the total stake in the protocol (from all validators including passive delegation) 
+// to the stake of a validator that a validator can achieve where the total stake of the validator is considered effective
+// (meaning the validator's total stake is used for caculating the lottery power or finalizer weight in the consensus).
+// Once a validator passes this bound, some of the validator's total stake no longer contribute to the validator's effective stake.
+// See the comment at the `delegated_capital_cap` type for the exact formula.
```

Comment:
It's probably also worth adding that delegators are prevented from increasing their delegation to the pool (via ConfigureDelegation) in such a way as would cause the pool to exceed the capital bound cap.